torchvision/utils.py

```diff
@@ -125,11 +125,17 @@ def save_image(
         tensor (Tensor or list): Image to be saved. If given a mini-batch tensor,
             saves the tensor as a grid of images by calling ``make_grid``.
         fp (string or file object): A filename or a file object
+        normalize (bool): This parameter has no effect and is deprecated. It should not be set.
         format(Optional):  If omitted, the format to use is determined from the filename extension.
             If a file object was used instead of a filename, this parameter should always be used.
         **kwargs: Other arguments are documented in ``make_grid``.
     """
 
+    if normalize != 'deprecated':
```

Comment:
I suggest to remove this completely, let the normalize be captured by `kwargs` and let it pass to the `make_grid` method call as intended.